src/components/App.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import './App.css';
+import Friends from './Friends';
+import BestFriends from './BestFriends';
+import AverageAge from './AverageAge';
+
+import Chance from 'chance';
+
+const chance = new Chance();
+
+
+
+export default class App extends Component {
+
+  constructor () {
+      super();
+      this.state = {
+        friends: []
+      }
+  }
+
+  componentWillMount() {
+    this.loadFriends();
```

Comment:
Awesome! Would be easier to change this into something that fetches friends from API or something similar :)